README.md

```diff
@@ -14,9 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-<h1><img alt="tool icon" src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/icon-deepsparse.png" />&nbsp;&nbsp;DeepSparse</h1>
 
-<h3>Neural network inference engine that delivers GPU-class performance for sparsified models on CPUs</h3>
```

Comment:
This h3 line as it reads needs to go back here under the product name as it's the tagline to the product: Neural network inference engine that delivers GPU-class performance for sparsified models on CPUs
I believe there will be a future effort to go through and update these product taglines.